<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1461</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>10</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartPushButton">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>110</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>220</y>
      <width>220</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLineEdit">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>280</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1709</x>
      <y>230</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter a number of rings:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
